app/views/questions/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<body>
+	<div id='all_questions'>
+		<p id='ui_title'>All Questions</p>
+			<% @questions.each do |question| %>
```

Comment:
This is hard to read because of the inconsistent and over-large indents. Think about setting an indent standard for your project and sticking to it.